logstash-core/benchmarks/src/main/java/org/logstash/benchmark/QueueBenchmark.java

```diff
@@ -87,8 +87,8 @@ private static Settings settings() {
             .capacity(256 * 1024 * 1024)
             .queueMaxBytes(Long.MAX_VALUE)
             .elementIOFactory(MmapPageIO::new)
-            .checkpointMaxWrites(50_000)
-            .checkpointMaxAcks(50_000)
+            .checkpointMaxWrites(1024)
```

Comment:
@jordansissel see last point in the PR description, I added this to verify that this change does not come with unexpected performance issues, so I moved the ack interval to the default here. Makes sense right?